index.js

```diff
@@ -5,5 +5,8 @@ export default (element, handler) => {
 	frame.style.cssText = CSS;
 	element.appendChild(frame);
 	frame.contentWindow.onresize = () => { handler(); };
-	return frame;
+	return () => {
+		iframe.contentWindow.onresize = null;
```

Comment:
Currently we return the frame, which gives you the DOM .remove() method for doing that.